tests/layers/CMakeLists.txt

```diff
@@ -114,3 +68,32 @@ add_vk_layer(wrap_objects ${WRAP_SRCS})
 
 set(TEST_SRCS test.cpp vk_layer_table.cpp vk_layer_extension_utils.cpp)
 add_vk_layer(test ${TEST_SRCS})
+
+# Run each .json.in file through the generator
+# We need to create the generator.cmake script so that the generator can be run at compile time, instead of configure time
+# Running at compile time lets us use cmake generator expressions (TARGET_FILE_NAME and TARGET_FILE_DIR, specifically)
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/generator.cmake" "configure_file(\"\${INPUT_FILE}\" \"\${OUTPUT_FILE}\")")
+foreach(CONFIG_FILE ${LAYER_JSON_FILES})
```

Comment:
I just used CONFIG_FILE because that was the old name, but you're right that it could be improved. I'd be tempted to use it BINARY_TARGET_NAME(S) as names, since that usually is the name of the layer binary's target, but that doesn't apply very well to metalayers.

I'm not sure what is best.

Reply:
Maybe just TARGET_NAMES.   But if none of these is obviously good, just stay with CONFIG_FILE and don't delay cuz of this.

Reply:
I went with TARGET_NAME(S). That's the best name I've seen here, so I'm going with that.